### ISSUE TITLE
Check to see file exists

### ISSUE THREAD

**Issue by Artemid** [Jintin/andle#4]
**andle** will fail if dependency path does not exist

file: `andle/sdk.py`
see the following lines

``` python
 40    ...
 41     find_dependency(data["dependency"], "", path + "/extras/android/m2repository")
 42     find_dependency(data["dependency"], "", path + "/extras/google/m2repository")
 43    ...
```

My configuration does not have "..../extras/google/m2repository"

It's a good idea to check if the path exists in `find_dependency` function before modifying anything in `data` 

e.g.

``` python
 52 
 53 def find_dependency(data, tag, path):
 54     if os.path.exists(path):
 55         no_dir = True
 56         for f in os.listdir(path):
 57             if (os.path.isdir(path + "/" + f)):
 58                 no_dir = False
 59                 find_dependency(data, tag + "/" + f, path + "/" + f)
 60         if no_dir:
 61             list = tag[1:].split("/")
 62             version = list.pop(len(list) - 1)
 63             name = list.pop(len(list) - 1)
 64             package = ".".join(list)
 65 
 66             update_value(version, data, package + ":" + name)
 67 
```

**Comment by Jintin** [comment]
Thank you very much.
Would you like to make pull request?

**Comment by Artemid** [comment]
Sure

**Comment by Jintin** [comment]
fix in #5 pull request
